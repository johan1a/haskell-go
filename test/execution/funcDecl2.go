package main;

func main() {
    var a int = 4
    func1(a)
}
func func1(x int) {
    println(2323)
    return func2(x)
}
func func2(y int) {
    println (y)
    return y
    
}
