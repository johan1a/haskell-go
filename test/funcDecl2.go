package main;

type yeah string
func main(){
}
