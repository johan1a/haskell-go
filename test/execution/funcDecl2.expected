2323
5
1
12345
